cmake/projects/OpenSSL/schemes/url_sha1_openssl_windows.cmake.in

```diff
@@ -66,6 +68,12 @@ else()
   set(do_ms do_ms.bat)
 endif()
 
+if(ASM_SUPPORT)
+  hunter_add_package(NASM)
```

Comment:
Please add a comment about `NASM_ROOT` variable so it will be easier to check dependency:
```cmake
hunter_add_package(NASM) # NASM_ROOT
```